index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("Ingrese su nombre:");
+alert("Hola usuario!");
```

Comment:
Se corrige el error en la alerta para que salude al usuario ingresado por prompt.

```suggestion
alert("Hola " + nombreUsuario );
```